GitStartHQ/graphql-engine
# [GQL-4] #7908 Hover on Relationships tab highlights unrelated cell

### Description


In Relationships tab under Data Table, hover state on over a relationship highlights the adjacent Object/Array relationship when they aren't related at all. This PR solves this problem and creates a solution where highlight is only over the current cell and empty cells can't be highlighted. 

### Changelog


### Affected components



### Related Issues




#7908

### Current Issue
https://user-images.githubusercontent.com/4939629/145628202-94088550-1851-44c5-9422-fe1b0dc051c8.mp4

### Solution and Design


The issue is solved by styling the table in such a way that only the current cell gets highlighted and adjacent cell doesn't get highlighted. Also, empty cell can't be highlighted.

https://user-images.githubusercontent.com/59878660/175192457-90b8c95c-cc1d-4764-8013-164448fb220f.mp4


### Steps to test and verify


1. Go to Table inside data section 
2. choose a table and go to relationships
3. If there are no relationship, create them first
4. Try to hover around the relationships cell and observe the result


### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  5. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  6. Logging
